app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from typing import List
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: List[Player]) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: List[Elf]) -> List[None]:
+    return [elf.play_elf_song() for elf in elves]
```

Comment:
There is no need to return a list of `None` in these methods, omit using the `return` statement and change your return type annotation accordingly.